File: OMWAYServices/OMWAY/src/main/java/com/example/omway/dto/payment/PaymentDto.java
package com.example.omway.dto.payment;

import com.example.omway.model.payment.PaymentMethod;
import lombok.Data;

@Data
public class PaymentDto {
    private Integer paymentId;
    private double total;
    private PaymentMethod paymentMethod;
    private Integer rideId;
}

File: OMWAYAppViews/OMWayApp/app/src/main/java/com/main/omwayapp/apirest/dto/payment/PaymentDto.kt
package com.main.omwayapp.apirest.dto.payment

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import kotlinx.serialization.Serializable

@Parcelize
@Serializable
data class PaymentDto(
    val paymentId: Int,
    val total: Double,
    val paymentMethod: String,
    val rideId:Int
) : Parcelable
